doc/snek.adoc

```diff
@@ -1105,10 +1145,10 @@ performs negation on numbers.(((–)))
 
 ``+`` _value_::
 When used as a unary prefix operator, the Unary Plus operator
-does nothing at all.(((+)))
+does nothing at all to a number.(((+)))
```

Comment:
actually, unary + can be applied to any value as it's simply ignored in the grammar. I guess that could be considered a bug somehow?

Reply:
Yeah, that's a bug. I want a syntax error when I say `+{1:2}` for sure.

Reply:
This would have to be checked at runtime (as Python3 does) to check the representation, not just catching a few cases in the parser where you use a constant. And that would require another op in the virtual machine, all to catch
errors in usage of an operator which otherwise doesn't do anything. Doesn't seem worth the space to me.